### ISSUE TITLE
No such method when running plugin

### ISSUE THREAD

**Issue by RobzInadE** [mjedynak/pit-idea-plugin#12]
Hi, I get this error when I try and run my pi tests:

`Exception in thread "main" java.lang.NoSuchMethodError: joptsimple.OptionParser.acceptsAll(Ljava/util/List;Ljava/lang/String;)Ljoptsimple/OptionSpecBuilder;
    at org.pitest.mutationtest.commandline.OptionsParser.<init>(OptionsParser.java:122)
    at org.pitest.mutationtest.commandline.MutationCoverageReport.main(MutationCoverageReport.java:36)
    at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
    at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
    at java.lang.reflect.Method.invoke(Method.java:497)
    at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)`

My configuration looks as follows:
![Image](http://i.imgur.com/GsJiZr6.png)

Any idea what's causing this?

**Comment by norbert-csibra-msm** [comment]
Same issue here, any solution?

**Comment by d215steinberg** [comment]
I'm getting it too.

For a single Eclipse instance, I am running Pitclipse successfully for one project, but I am getting the error for the other.  The failing project was built with Java 1.7, while the succeeding one was built with 1.8.  I re-built the failing project with 1.8, but that did not help.

**Comment by jiteshvassa** [comment]
I've recently started to see this too.  I'm running Eclipse Version: Mars.2 Release (4.5.2), Build id: 20160218-0600

Anyone have an update on what's causing it?

Here's the full stack:

```
Exception in thread "main" java.lang.NoSuchMethodError: joptsimple.OptionParser.acceptsAll(Ljava/util/List;Ljava/lang/String;)Ljoptsimple/OptionSpecBuilder;
	at org.pitest.mutationtest.commandline.OptionsParser.<init>(OptionsParser.java:122)
	at org.pitest.mutationtest.commandline.MutationCoverageReport.main(MutationCoverageReport.java:36)
	at org.pitest.pitclipse.pitrunner.PitRunner.runPIT(PitRunner.java:46)
	at org.pitest.pitclipse.pitrunner.PitRunner.main(PitRunner.java:25)
```

**Comment by shaun-w** [comment]
I'm getting the same error as well, and can't find anyway to get it working. Looks like it cannot find the joptsimple library... it is in my project dependencies, but looks like the plugin cannot see it. Any ideas how to get it to pick up the library?

**Comment by pierre-jenchen** [comment]
I run into this issue, too. Is there a workaround for that?

**Comment by ehdez73** [comment]
Same issue here.

**Comment by dtpmalone** [comment]
I have the same issue

**Comment by giggluigg** [comment]
+1

**Comment by mjedynak** [comment]
fixed in 1.3.8
